data/mods/Aftershock/recipes/recipes.json

```diff
@@ -605,5 +605,44 @@
       [ [ "afs_material_1", 6 ] ],
       [ [ "pipe", 3 ] ]
     ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "scrapbow",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "archery",
+    "skills_required": [ [ "fabrication", 3 ] ],
+    "difficulty": 3,
+    "autolearn": true,
+    "reversible": true,
+    "time": "4 h",
+    "qualities": [ { "id": "SAW_M_FINE", "level": 1 } ],
+    "using": [ [ "welding_standard", 3 ] ],
+    "proficiencies": [ { "proficiency": "prof_metalworking" }, { "proficiency": "prof_welding_basic" } ],
+    "components": [ [ [ "afs_material_1", 8 ] ], [ [ "afs_circuitry_2", 1 ] ], [ [ "afs_energy_storage_1", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "activity_level": "MODERATE_EXERCISE",
+    "result": "scrapcrossbow",
+    "category": "CC_WEAPON",
+    "subcategory": "CSC_WEAPON_RANGED",
+    "skill_used": "rifle",
+    "skills_required": [ [ "fabrication", 3 ] ],
+    "difficulty": 3,
+    "autolearn": true,
+    "reversible": true,
+    "time": "4 h",
+    "qualities": [ { "id": "SAW_M_FINE", "level": 1 }, { "id": "SCREW_FINE", "level": 1 }, { "id": "WRENCH", "level": 2 } ],
+    "using": [ [ "welding_standard", 3 ] ],
+    "proficiencies": [ { "proficiency": "prof_metalworking" }, { "proficiency": "prof_welding_basic" } ],
+    "components": [
+      [ [ "afs_material_1", 6 ] ],
+      [ [ "afs_circuitry_2", 3 ] ],
+      [ [ "afs_energy_storage_1", 2 ] ],
+      [ [ "afs_magnet_1", 1 ] ]
```

Comment:
I'm thinking it would require four magnets to work in two sets in a crossbow design?

Reply:
That can be fixed, give me a moment.